base/intfuncs.jl

```diff
@@ -96,6 +96,8 @@ end
 
 Least common (positive) multiple (or zero if any argument is zero).
 The arguments may be integer and rational numbers.
+``a`` is a multiple of ``b`` if there exists an integer ``m`` such
+that ``b*m=a``.
 
 ``a`` is a multiple of ``b`` if there exists an integer ``m`` such
 that ``a=mb``.
```

Comment:
This note is already present; it previously looked like it wasn't because the base commit was out of date.